pkg/config/setup/apm.go

```diff
@@ -84,6 +84,15 @@ func setupAPM(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("apm_config.instrumentation.enabled_namespaces", []string{}, "DD_APM_INSTRUMENTATION_ENABLED_NAMESPACES")
 	config.BindEnvAndSetDefault("apm_config.instrumentation.disabled_namespaces", []string{}, "DD_APM_INSTRUMENTATION_DISABLED_NAMESPACES")
 	config.BindEnvAndSetDefault("apm_config.instrumentation.lib_versions", map[string]string{}, "DD_APM_INSTRUMENTATION_LIB_VERSIONS")
+	// Note(stanistan): The flag "DD_APM_INSTRUMENTATION_VERSION"
+	//                  will remain undocumented for the duration of the beta.
+	//                  We intend to only switch back to v1 if beta customers have issues
+	//                  to troubleshoot. v1 will be dropped (and possibly the flag)
+	//                  in 7.58 or 7.49 (this is going out in 7.57).
```

Comment:
Why prefix with `Note(stanistan):` ? 😄‏

Reply:
So people can find me if they need to :)